evap/staff/importers.py

```diff
@@ -278,7 +280,7 @@ def consolidate_enrollment_data(self):
 
     def check_evaluation_data_correctness(self, semester):
         for evaluation_data in self.evaluations.values():
-            already_exists = Evaluation.objects.filter(course__semester=semester, name_de=evaluation_data.name_de).exists()
+            already_exists = Course.objects.filter(Q(semester=semester) & (Q(name_en=evaluation_data.name_en) | Q(name_de=evaluation_data.name_de))).exists()
             if already_exists:
                 self.errors.append(_("Course {} does already exist in this semester.").format(evaluation_data.name_en))
```

Comment:
I suggest doing the following so that a user does not get confused because we're printing back the name in a language that is not actually the problem:
```python
if Course.objects.filter(semester=semester, name_en=evaluation_data.name_en).exists():
    self.errors.append(_("Course {} does already exist in this semester.").format(evaluation_data.name_en))
if Course.objects.filter(semester=semester, name_de=evaluation_data.name_de).exists():
    self.errors.append(_("Course {} does already exist in this semester.").format(evaluation_data.name_de))
```

I think one more line of code for this is reasonable.